CMakeLists.txt

```diff
@@ -20,8 +20,8 @@ option(BUILD_WINDOWS_MIDDLEWARE_LIB "Build the Windows OpenSSH middleware librar
 option(BUILD_TESTS "Build tests" ON)
 
 # Configurable OpenSSH security key API version
-set(SK_API_VERSION 7 CACHE STRING "OpenSSH security key API version to target")
-list(APPEND VALID_SK_API_VERSIONS 5 7)
+set(SK_API_VERSION 9 CACHE STRING "OpenSSH security key API version to target")
```

Comment:
Because Debian stable still uses OpenSSH 8.4, I'd prefer that we keep sk-api API version 7 as the default. If someone needs to use a newer version of OpenSSH, it's easy to configure CMake to use the newer API version since it's explicitly documented in the README.

Reply:
Fair enough.